lib/convert.js

```diff
@@ -211,6 +211,10 @@
         var convertProc = (_shouldUseFlite(settings, binaryPaths) ?
                            spawn(binaryPaths.flite, args) :
                            spawn(binaryPaths.convert, args));
+        
+        if (_shouldUseFlite(settings, binaryPaths)) {
+            logger.log("Using Flite for image encoding");
+        }
```

Comment:
the else is image magic and that's the default for everything that's not export as so it might get noisy